src/components/Dashboard/Dashboard.tsx

```diff
@@ -32,9 +38,39 @@ export const Dashboard = withRouter(props => {
   /**
    * Set token in token context
    */
-  tokenContext.setToken(getCookie("sessionToken"));
+  useEffect(() => {
+    /**
+     * Set token in token context
+     */
+    tokenContext.setToken(getCookie("sessionToken"));
+    console.log("setToken called!", tokenContext.token);
+  }, []);
 
-  console.log("setToken called!", tokenContext.token);
+  useEffect(() => {
+    console.log("Enter use effect with toker", tokenContext.token);
+    if (tokenContext.token) {
+      const url =
+        customWindow._env_.IO_ONBOARDING_PA_API_HOST +
+        ":" +
+        customWindow._env_.IO_ONBOARDING_PA_API_PORT +
+        "/profile";
+      fetch(url, {
```

Comment:
do we have a plan (pivotal task) to replace these fetch calls with the generated types + API client (from the OpenAPI specs) like in the other projects ? You may take the io-backend package.json as a reference (of the io-utils cli): 

https://github.com/teamdigitale/io-backend/blob/master/package.json#L20

and here is the API client:

https://github.com/teamdigitale/io-backend/blob/master/src/clients/api.ts

The concept is to "embed" all API calls into an API client which use the Typescript types generated from the OpenAPI specs. Moreover in this way you  don't have to extract the token and the baseUrl for every api call (saves some lines).

Reply:
i will check this but i opened a specific story for this task, so for this pr I would leave as it is

Reply:
@d-al-ibm @mccalv this is mandatory and blocking for all projects, we can't rely on manually maintained clients

Reply:
as a side note, we've discussed this with @d-al-ibm and @alexgpeppe: until https://github.com/aymen94/io-utils-build is completed, using OpenAPI 3.x is a blocker for the backend (io-onboarding-pa-api) as well.